capstone-frontend/src/styles/Groups.css

```diff
@@ -1,36 +1,95 @@
+#add-members {
+  margin-bottom: 20px;
+}
+
 .admin-button {
   margin-top: 20px;
 }
 
+.assignment-card {
+  text-align: left;
+  border: 1px solid grey;
+  box-shadow: 3px 3px 3px #888888;
+  margin-bottom: 10px;
```

Comment:
nit: alphabetize specific styles

Reply:
Will do! Once I merge PR #91 into master and merge this with origin/master it will re-alphabetize correctly as I'm fixing this in that PR.